### DISCUSSION TITLE
How to disable cloning ssh session when opening new tab?

### DISCUSSION THREAD

**Discussion by Yevgnen** [kovidgoyal/kitty#6707] [Q&A] [answered]
I have the following config:

```
shell_integration no-cursor no-title
map kitty_mod+t launch --cwd=current --type=tab
```

Shell integration is great except that I don't want it to clone ssh session when I do `kitty_mod + t`. How can I disable this part of the feature?

**Comment by kovidgoyal** [comment, marked as answer]
Dont use --cwd=current
make a different mapping without it and use that when you dont want to
preserve the working environment.
